compiler/dyno/include/chpl/uast/Variable.h

```diff
@@ -98,6 +99,25 @@ class Variable final : public VarLikeDecl {
   bool isConfig_;
   bool isField_;
 
+  void setInitExprForConfig(AstNode* ie) {
```

Comment:
IMO it would be nicer for this take in an `owned<AstNode> ie` (i.e. by value) since it will assume ownership of it. Would simplify the below code as well (removing `toOwned`).